vendor/.htaccess

```diff
@@ -1,8 +0,0 @@
-<IfVersion < 2.4>
```

Comment:
Please revert the `vendor/.htaccess` removal

Reply:
I don't remember deleting this file. It must have been an accident. I have just put it back.